components/nexus-repository/src/main/java/org/sonatype/nexus/repository/group/GroupFacetImpl.java

```diff
@@ -77,7 +78,7 @@ public boolean member(final Repository repository) {
   @Override
   @Guarded(by = STARTED)
   public List<Repository> members() {
-    List<Repository> members = Lists.newArrayListWithCapacity(memberNames.size());
+    List<Repository> members = new ArrayList<>(memberNames.size());
```

Comment:
The javadoc for newArrayListWithCapacity says that it's deprecated as of Java 7, since the diamond syntax does the same job. Same with newArrayList(), newArrayList(E.. elements), and several other Lists methods.